### DISCUSSION TITLE
USWDS - Feature: Star rating component

### DISCUSSION THREAD

**Discussion by ryanwoldatwork** [uswds/uswds#6273] [Proposals]
### Is your feature request related to a problem? Please describe.

As a Form Manager,
I'd like to implement a star-rating component
So that I can get 1-5 feedback on one or more questions.

### Describe the solution you'd like

A USWDS styled component that looks similar to

![Image](https://github.com/user-attachments/assets/c7bff13b-3a88-40b5-b5fe-01e9a538f9f7)

Reference:
https://www.w3.org/WAI/tutorials/forms/custom-controls/

### Describe alternatives you've considered

5 radio button likert scale

### Additional context

A version of a 5-star rating is currently implemented in Touchpoints.
Touchpoints plans to phase this out in FY2025 Q1 or Q2, unless USWDS decides to consider moving forward with it.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Feature+Request%22) for duplicate feature requests.

**Comment by amycole501** [comment]
I spoke with Ryan about using the [USWDS range slider ](https://designsystem.digital.gov/components/range-slider/)in a similar way to how W3 features it as a [satisfaction rating component](https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-rating/).

Ryan is already working with the range slider and adding stops/increments into it to create a five "star" rating system. He'll report back an option for us to perform accessibility testing on in the coming weeks.
